pyteal/ast/abi/tuple.py

```diff
@@ -198,9 +198,9 @@ def _index_tuple(
         if offset == 0:
             # This is the first and only value in the tuple, so decode all of encoded
             return output.decode(encoded)
-        # This is the last value in the tuple, so decode the substring from start_index to the end of
-        # encoded
-        return output.decode(encoded, start_index=start_index)
+        elif all(not x.is_dynamic() for x in value_types):
```

Comment:
nit: `elif` not required because the previous if will always return.
```suggestion
        if all(not x.is_dynamic() for x in value_types):
```

Reply:
[8fb2e33](https://github.com/algorand/pyteal/pull/663/commits/8fb2e33f8448ddd58668855a2a71df0982a7678b)